conversions.py

```diff
@@ -5,4 +5,10 @@ def miles_to_km(num):
     return float(num)*1.60934
 
 def in_to_cm(num):
-    return num*2.54
\ No newline at end of file
+    return num*2.54
+
+def miles_to_yards(num):
+    return num*176
```

Comment:
You messed up the conversion here by a factor of 10.
```suggestion
    return num*1760
```